### DISCUSSION TITLE
There is a problem after submitting the supporting documents, so how can I apply again?

### DISCUSSION THREAD

**Discussion by kbh0218** [community/community#123566] [GitHub Education]
### Hi everyone,

I re-apply after receiving an e-mail saying that I cannot confirm my academic affiliation. But when I re-apply, my original GitHub account is logged in and I cannot apply. How can I re-apply because there is a problem after submitting the supporting documents?

**Comment by AmimiHamza** [comment]
you  just have to send again an image of an academic paper , make sure you pick a different one from the previous one (you can just change the size)
